docker/docker.mk

```diff
@@ -136,8 +138,10 @@ else
 	fi
 endif
 else
-	-$(DOCKER) buildx create --use --name daprbuild
-	-$(DOCKER) run --rm --privileged multiarch/qemu-user-static --reset -p yes
+	-$(DOCKER) run --privileged --rm tonistiigi/binfmt:qemu-v7.0.0 --install all
```

Comment:
This must happen before creating the build in buildx. Had to switch to a multiarch image (qemu-user-static is amd64 only). Lastly, pinned to the current latest tag so we have control over upgrades instead of having :latest to break the build in the future.